### PR TITLE
Update video demos for Meta Search

### LINKED ISSUE #418: Recreate Meta Search marketing page documentation with Openverse screenshots
The gifs on https://wordpress.org/openverse/meta-search currently use old 'footage' from CC Search. This should be replaced with Openverse.

### PR DESCRIPTION
**Pull request by dhruvkb**
## Fixes

Fixes #418 by @zackkrida 

## Description

This PR
- records and updates <s>video</s> image demos for Openverse replacing the old CC videos
	- uses <s>WebM for smoother playback and</s> PNG for smaller file sizes
- updates the language to be future-proof for when audio becomes a first-class media type
	- also replaces hard-coded text for buttons and sections with references to the actual buttons and sections

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/locales/po-files/openverse.pot [hunk 1]
```diff
@@ -1518,103 +1515,105 @@ msgctxt "meta-search-page.content-types"
...
 msgctxt "meta-search-page.audio-video.title"
-msgid "How does the Meta Search feature work for Audio and Video?"
+msgid "How does the Meta Search feature work for non-indexed media types?"
 msgstr ""
 
-#: src/pages/meta-search.vue:50
+#: src/pages/meta-search.vue:65
 msgctxt "meta-search-page.audio-video.content"
-msgid "Openverse does not yet support built-in audio and video search. For Audio and Video, simply click on the appropriate tab, and you’ll see what sources you can find CC-licensed content on."
+msgid "Openverse does not yet support built-in search for some media types, such as videos. For these media types, simply click on the appropriate tab, and you’ll see what sources you can find CC-licensed content on."
```
Comment (sarayourfriend) [inline src/locales/po-files/openverse.pot:1582]:
```suggestion
msgid "Openverse does not yet support built-in search for some media types, such as videos. For these media types, simply click on the appropriate tab and you’ll see sources where you can find CC-licensed content."
```

**Review by sarayourfriend — CHANGES_REQUESTED** [review]
Thanks for updating this and reducing the filesize, that's a really big deal!

The only thing is that these videos are way too tall for many screen sizes. The originals were not very tall and easily fit on most screen sizes.

On mobile it is too difficult to read anything because the video includes so much detail that all the text is blurry and small. The original gifs were not _much_ better but they were a little better.

We should also add the `loop="true"` attribute to these videos so that they replay like the gifs do.

**Comment by dhruvkb** [comment]
@sarayourfriend from a usability and accessibility perspective, would images be more preferable than a video or a GIF? My understanding here is that in a short video without commentary, it's very hard to grasp what actually happened whereas an image can have both alt text and drawn-on markers, making the content quite clear.

File: src/locales/po-files/openverse.pot (hunk 1 above)
Comment (dhruvkb) [reply src/locales/po-files/openverse.pot]: Fixed in 10f9fff.

**Comment by sarayourfriend** [comment]
Hmm, probably an image would be better. When I see a video or GIF like this playing on a page I tend to skip right over it, but I don't know if that's how most people behave. Maybe @panchovm would know.

**Comment by panchovm** [comment]
I would use images instead of GIFs or videos. The sequences recreate the whole flow unnecessarily. Having screenshots of the pages where Meta Search areas are centered is a solution that solves the need. Also agree with @sarayourfriend, the crop can be a laptop-screen ratio.

For the last part, where we explain the why of this feature, I will get rid of the image. It doesn't add more value than the current description.

**Review by obulat — CHANGES_REQUESTED** [review]
The screenshot alt text does not make it clear what it contains. Something like "Screenshot showing the meta search form with buttons to search for openly-licensed media of unsupported type on other sites" would be better for a11y.

**Comment by sarayourfriend** [comment]
> The screenshot alt text does not make it clear what it contains. Something like "Screenshot showing the meta search form with buttons to search for openly-licensed media of unsupported type on other sites" would be better for a11y.

Just a nit pick, but I think "screenshot showing" is redundant in alt text. At least I know it's recommended to not say "picture of ..." because the screen reader will already announce that it is an image. Maybe screenshot is extra useful information in this case though? 🤔

**Comment by dhruvkb** [comment]
To be fair, we could entirely make the image invisible to screen-readers for these reasons: 
- the information is already covered in the text, and the image is not needed to understand it
- the image adds nothing more than visual help, so it's zero value add for visually impaired users

**Comment by sarayourfriend** [comment]
@dhruvkb Nice thinking! Makes sense to me!

**Review by obulat — APPROVED** [review]

**Pull request merged**